### DISCUSSION TITLE
Uncomfortable snapshot-only

### DISCUSSION THREAD

**Discussion by bangetto** [modrinth/.github#49] [Feature requests]
1.20 is curently in snapshot as we all know, but projects wich are "_1.20-only_" are appiering and in those projects if you click on the `Filter Version` button it will show nothing and you need to manually press the `Include Snapshots` button.

You probably gussed where I'm going with this, so make the "_snapshot-only_" projects have the `Include Snapshots` button automaticly turned on, or an option in the project/user setting for that.

**Comment by Andre601** [comment]
I don't this would be good.

A normal user usually only looks for versions compatible with release versions of mc and not for snapshots.

If they do want to search snapshot-compatible versions should a little effort be given.

Also, including snapshots by default for mods supporting these may cause dozens of entries,making it hard tosearch.

**Reply by Gaming32** [reply]
They're referring to the versions page, I think.

**Reply by bangetto** [reply]
Yeah, I'm refering to the version page.

**Reply by Andre601** [reply]
It was obvious to me that they refer to the versions page of a resource.

My point stands that the list could become unreasonably large if snapshots are included by default just because the project supports them.

**Reply by bangetto** [reply]
I didn't meant default, I meant as an optional setting or like that modrinth detects if the project is only for snapshots and if so it's enables the snapshot filter automaticly

**Comment by triphora** [comment]
This suggestion is a case of the XY problem, I believe; the real fix would be to show snapshot versions when you're typing a specific version in

**Reply by bangetto** [reply]
This is a good idea too, but I think the average user don't realize that you can type there, but if my original suggestion didn't make it, this has to it.

Edit: I have no idea btw why you need to press a button to type in a snapshot

**Reply by Minenash** [reply]
Honestly, over time I also forgot you could type there
